### DISCUSSION TITLE
Open Script Kit in second desktop window

### DISCUSSION THREAD

**Discussion by georgefairbairn** [johnlindquist/kit#1377] [Ideas]
Currently, if I try and launch Script Kit from a second desktop window on Mac OS it doesn't launch. It will only launch from my primary desktop.

It would be nice to be able to launch Script Kit from any desktop!

**Comment by johnlindquist** [comment]
@georgefairbairn this should be fixed in the 2.0 Release candidate:

https://github.com/johnlindquist/kit/discussions/1365
